Realm/ObjectServerTests/RLMSyncTestCase.mm

```diff
@@ -314,7 +324,7 @@ - (void)waitForDownloadsForUser:(RLMSyncUser *)user
         XCTFail(@"Download waiter did not queue; session was invalid or errored out.");
         return;
     }
-    [self waitForExpectations:@[ex] timeout:20.0];
+    [self waitForExpectations:@[ex] timeout:60.0];
```

Comment:
What's this PR's effect on the total runtime of the test suite? The ROS tests are already pretty lengthy. Perhaps we should be conditionally running tests with degraded network connectivity? Much like we do with `build.sh verify-osx-encryption`.